utils/EVMRuntime.js

```diff
@@ -825,6 +829,44 @@ module.exports = class EVMRuntime {
   }
 
   async handleCALL (runState) {
+
+    const gasLimit = runState.stack.pop();
+    const toAddress = runState.stack.pop();
+    const value = runState.stack.pop();
+    const inOffset = runState.stack.pop();
+    const inLength = runState.stack.pop();
+    const outOffset = runState.stack.pop();
+    const outLength = runState.stack.pop();
+
+    const data = this.memLoad(runState, inOffset, inLength);
+    const funcSig = this.getFuncSig(data);
+
+    if (funcSig === '0x22334455') {
+      this.subMemUsage(runState, outOffset, outLength);
+
+      if (gasLimit.gt(runState.gasLeft)) {
+        gasLimit = new BN(runState.gasLeft);
+      }
+
+      const color = '0x' + toAddress.toString(16, 40);
+      const to = utils.bufferToHex(data.slice(4, 24));
+      const from = utils.bufferToHex(runState.caller);
+      const amount = utils.bufferToHex(data.slice(36, 68));
+
+      const success = runState.tokenBag.transfer(color, from, to, amount);
+
+      if (success) {
+        runState.stack.push(new BN(1));
```

Comment:
add comment here if `1` is success or error

Reply:
or use constant

Reply:
FIXED.